test/conftest.py

```diff
@@ -132,3 +134,17 @@ def cartridge_app(request, cartridge_app_session):
             bootstrap_vshard = params["bootstrap_vshard"]
     cartridge_app_session.truncate(bootstrap_vshard=bootstrap_vshard)
     return cartridge_app_session
+
+
+@pytest.fixture(scope="function")
+def tcs(request, tmp_path):
+    test_app_path = os.path.join(utils.path_to_tcs_dir, "config.yaml")
+    inst = utils.TarantoolTestInstance(test_app_path, utils.path_to_tcs_dir, "", tmp_path)
+    inst.start(connection_test=True,
+               connection_test_user="client",
+               connection_test_password="secret",
+               instance_name="instance-001",
+               instance_host="localhost",
+               instance_port="3303")
```

Comment:
As with the path to the configuration file, these parameters must be configured from a specific test if it is possible.

Reply:
Made it possible by using special fixture `fixture_params()` which can set values globally in dictionary to provide it to another fixture. So now I can to configure it directly from tests.